src/analyses/apron/apronPriv.apron.ml

```diff
@@ -834,49 +834,19 @@ module PerMutexMeetPrivTID (Cluster: ClusterArg): S  = functor (AD: ApronDomain.
 struct
   open CommonPerMutex(AD)
   include MutexGlobals
-  include ConfCheck.RequireThreadFlagPathSensInit
 
   module NC = Cluster(AD)
   module Cluster = NC
   module LAD = NC.LAD
 
-  module LLock =
-  struct
-    include Printable.Either (Locksets.Lock) (CilType.Varinfo)
-    let mutex m = `Left m
-    let global x = `Right x
-  end
-
-  (* Map from locks to last written values thread-locally *)
-  module L = MapDomain.MapBot_LiftTop (LLock) (LAD)
-
-  module LMust = struct
-    include SetDomain.Reverse (SetDomain.ToppedSet (LLock) (struct let topname = "All locks" end))
-    let name () = "LMust"
-  end
+  include PerMutexTidG(struct
+      let exclude_not_started () = GobConfig.get_bool "ana.apron.priv.not-started"
+      let exclude_must_joined () = GobConfig.get_bool "ana.apron.priv.must-joined"
+    end)(LAD)
 
   module D = Lattice.Prod3 (W) (LMust) (L)
```

Comment:
This should already come from `include PerMutexTidG`.

Reply:
What exactly are you referring to here?

Reply:
I mean the included `CommonPriv.PerMutexTidCommon` already contains `module D = Lattice.Prod3 (W) (LMust) (L)`.